MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+namespace MauiAppBasic.MSALClient
+{
+    /// <summary>
+    /// This is a wrapper for PCA. It is singleton and can be utilized by both application and the MAM callback
+    /// </summary>
+    public class PCAWrapper
+    {
+        /// <summary>
+        /// This is the singleton used by consumers
+        /// </summary>
+        public static PCAWrapper Instance { get; private set; } = new PCAWrapper();
+
+        internal IPublicClientApplication PCA { get; }
+
+        internal bool UseEmbedded { get; set; } = false;

```

Comment:
Not sure I get the point of `internal` here. It should either be private or public.

Reply:
It can not be private as it is set by MainPage. This property is not for public consumption and is being used only internally, so kept it internal. There is no need to make it public.